Bottom: 45f554e00a38d3b2dc1972bfed7771ff6816b165
Top:    041c3c8e83fc413327d4c41cea77d100810a8443
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 15:47:09 +0000

Refresh of change-calculation-for-how-man

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 6931651..0882b5c 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -606,7 +606,8 @@ do_hole (const BoxType *b, void *cl)
    * hidgl_fill_circle to draw this contour.
    */
   if (curc->is_round) {
-    double slices = calc_slices (curc->radius / info->scale, 2 * M_PI);
+    double slices = M_PI * 2 * curc->radius /
+                    info->scale / PIXELS_PER_CIRCLINE;
     if (slices < curc->Count) {
       hidgl_fill_circle (curc->cx, curc->cy, curc->radius, info->scale);
       return 1;
